sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,907 @@
+
+## BTC.b
+
+This is a closed membership system.  It is operated by eight wardens (relayers)
+and a private codebase, secured by an SGX enclave. The SGX enclave only
+processes transfers when it receives approvals from at least 6 of the 8 Warden nodes.
+
+## LBTC
+
+This is a closed membership system.  Over 50 members of Liquid Federation manage multi-sig
+contract.  Only federation members can process withdraws.
+
+## tBTC
+
+This is an open membership system, where the BTC is managed by a
+rotating set of randomly selected nodes which manage a threshold
+wallet.  However, it incurs fork risk because it does not operate atop Bitcoin.
+
+sBTC implements a state of the art solution whereby network validation is open
+and permissionless -- anyone can run a validator to secure the sBTC network
+ -- and where transactions settle to Bitcoin which benefit from 100% of
+Bitcoin's reorg resistance.
+
+# Backwards Compatibility
+
+This SIP would be a hard fork.  It would introduce a boot contract `.sbtc` and
+would alter the semantics of PoX such that token unlocks and PoX payouts could
+be halted based on pending sBTC operations.
+
+However, the act of Stacking would remain the same as before.
+
+# Activation
+
+This SIP can activate in one of the following ways:
+
+## Concurrent with SIP-021
+
+If the implementation of this SIP is ready and merged to the `master` branch
+of the Stacks blockchain by the time SIP-021 activates, then this SIP also
+activates.
+
+## As a Future Hard Fork
+
+If this SIP does not ship with SIP-021, then a follow-on hard fork will be
+needed to activate it.  To activate this SIP, a Stacker vote will need to be
+taken to assess how they feel about running and maintaining the sBTC wallet, in
+addition to their block-signing responsibilities in SIP-021.
+
+In order for this SIP to activate, the following criteria must be met by the set
+of Stacked STX:
+
+* At least 80 million Stacked STX must vote _at all_ to activate this SIP.  This
+  number is chosen because it is more than double the amount of STX locked by
+the largest Stacker at the time of this writing (reward cycle 69).
+
+* Of the Stacked STX that vote, at least 80% of them must vote "yes."
+
+The act of not voting is the act of siding with the outcome, whatever it may be.
+We believe that these thresholds are sufficient to demonstrate interest from
+Stackers -- Stacks users who have a long-term interest in the Stacks
+blockchain's succesful operation -- in performing this upgrade.
+
+### How To Vote
+
+If a user is Stacking, then their STX can be used to vote in one of two ways,
+depending on whether or not they are solo-stacking or stacking through a
+delegate.
+
+The user must be Stacking in any cycle up to and including cycle XXX.  Their vote
+contribution will be the number of STX they have locked.
+
+#### Solo Stacking
+
+The user must send a minimal amount of BTC from their PoX reward address to one
+of the following Bitcoin addresses:
```

Comment:
Are the BTC sent to these addresses considered unspendable?

Reply:
Well, if they are spendable, you're more than welcome to claim them yourself ;)

Reply:
@jcnelson I will take that as a "yes"? The problem with these unspendable "burn addresses" is that they pollute the UTXO set with UTXOs that can never be spent, so they just bloat the UTXO set forever for no good reason. We have a better solution for burning BTC, it is by specifying an OP_RETURN output in a transaction and assigning an amount of BTC to be spent in this output. The BTC is then provably burnt but it does not bloat the UTXO set. (This is the difference between "the simple way" and "the meaningful way" defined [here](https://medium.com/@alcio/how-to-destroy-bitcoins-255bb6f2142e))

Reply:
@jcnelson is this part still relevant with the intent of putting the burned BTC from the prepare phase into the wallet's warchest?